### PR TITLE
WL-488: Flash Messages expired in bulk purchase

### PR DESCRIPTION
**Pull request by malikshahzad228**
Amount message is still appearing in some cases.

[WL-488](https://openedx.atlassian.net/browse/WL-488)

### CODE REVIEW

**Comment by malikshahzad228** [comment]
@mattdrayer @asadiqbal08 @saleem-latif FYI

**Comment by mattdrayer** [comment]
👍

File: ecommerce/extensions/basket/views.py [hunk 1]
```diff
@@ -133,9 +133,11 @@ def get_context_data(self, **kwargs):
             except (ConnectionError, SlumberBaseException, Timeout):
                 logger.exception('Failed to retrieve data from Course API for course [%s].', course_key)
 
-            # Set to true if any course in basket has bulk purchase scenerio
+            # Set to true if any course in basket has bulk purchase scenario
             if line.product.get_product_class().name == ENROLLMENT_CODE_PRODUCT_CLASS_NAME:
                 is_bulk_purchase = True
+                # Iterate on message storage so messages marked as read
```
Comment (saleem-latif) [inline ecommerce/extensions/basket/views.py]: nit: Iterate on message storage so all messages are marked as read

**Comment by saleem-latif** [comment]
@malikshahzad228 just one nit, other than that LGTM 👍

**Pull request merged**